haystack/modeling/training/base.py

```diff
@@ -251,8 +250,8 @@ def train(self):
                 vocab_size1=len(self.data_silo.processor.query_tokenizer),
                 vocab_size2=len(self.data_silo.processor.passage_tokenizer),
             )
-        elif not isinstance(
-            self.model.language_model, DebertaV2
+        elif (
+            not self.model.language_model.name == "debertav2"
```

Comment:
Why not `self.model.language_model.name != "debertav2"`?

Reply:
No good reason, let me change this :smile: